experiments/models.py

```diff
@@ -103,6 +104,15 @@ def to_dict(self):
 
     def to_dict_serialized(self):
         return json.dumps(self.to_dict(), cls=DjangoJSONEncoder)
+    
+    def reset_counters(self):
+        Counters().reset_prefix(self.name)
+    
+    def delete(self, reset_counters=True, *args, **kwargs):
+        if reset_counters:
+            self.reset_counters()
+        return super(Experiment, self).delete(*args, **kwargs)
```

Comment:
Do we not need to tidy up the enrollments as well?

Reply:
Enrollments have `on_delete=models.CASCADE`

Reply:
That could be 1M+ rows...... do we want to rely on cascade or manually
iterate through... I dont really have a good answer to this, but thought
I'd flag it.